04-halstead-complexity/04-halstead-complexity.tex

```diff
@@ -92,9 +92,11 @@
 \end{itemize}
 }
 
-\pitch{
-\pptQuote{bill-curtis.jpg}{In studying some error data provided us by Rome Air Development Center, Phil Milliman and I found Halstead's metric a remarkably accurate predictor of delivered bugs in a system developed with modern programming practices and tools}{\textit{Program Complexity and Software Errors: \newline A Front End for Reliability}, \newline Bill Curtis (father of SEI CMM), 1979}
-}
+\qte
+  [Bill Curtis]
+  {{bill-curtis.jpg}
```

Comment:
@awggeez it's a typo: extra opening curled bracket

Reply:
typo corrected